components/molecules/client/bookmark-btn/component.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import { Btn } from '@/components/atoms/btn';
+
+import { BookmarkIcon as BookmarkIconOutline } from '@heroicons/react/24/outline';
+import { BookmarkIcon as BookmarkIconSolid } from '@heroicons/react/24/solid';
+
+import React from 'react';
+
+interface Props extends React.ComponentProps<typeof Btn>{}
```

Comment:
Props doesn't need to include children or variant

Reply:
Extending props, the component still requires you to pass children and variant. You can use typescript Omit to omit those.